src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -160,6 +160,19 @@ def _get_preprocess_shape(self, oldh: int, oldw: int, long_side_length: int) ->
         return (newh, neww)
 
 
+class ResizetoShortEdge(ResizetoLongestEdge):
```

Comment:
It's parent class `ResizetoLongestEdge` which was previously implemented for visual prompting is dependent on torchvision ops, but torchvision ops showed slow performance.
I think `ResizetoLongestEdge` should be deprecated and using `Resize` implemented with mmdet components looks better. (will be updated)
And this `ResizetoShortEdge` should be also updated along with `Resize`.
`Resize` already supports `keep_ratio` and this option works the same with `ResizetoLongestEdge`.
What do you think?

Reply:
Yes, I agree.
However, `Resize` imported into OTX still has the issue of being slow on large data. - https://github.com/openvinotoolkit/training_extensions/issues/3379

This looks like it needs to be optimized and unnecessary classes removed.
I'll replace this with `Resize (keep_ratio=True)` and compare the two scores.

Reply:
Thanks - Done https://github.com/openvinotoolkit/training_extensions/pull/3380/commits/13a3491503e9984ad843c1ce08649601c64da06c

Reply:
When resizing an image with (3, 5) to (10, 10), 
`Resize (keep_ratio=True)` and `ResizetoLongestEdge` resize it to (6, 10) based on the longest edge.
`ResizetoShortEdge` resizes it to (10, 16) based on the shortes edge.
It is difference, please refer it.